### PR TITLE
Add a CLI option to change output filenames

### PR DESCRIPTION
**Pull request by iwata**
Add `--underscore` option for generator.
If you enable this option, the generated filenames will be word-separated and underscores will be inserted.

### CODE REVIEW

File: test/testmodels/customtypes/compositeprimarykey.yo.go [hunk 1]
```diff
@@ -371,10 +371,10 @@ func ReadCompositePrimaryKeysByZError(ctx context.Context, db YORODB, keys spann
 	return res, nil
 }
 
-// FindCompositePrimaryKeysByZYError retrieves multiple rows from 'CompositePrimaryKeys' as a slice of CompositePrimaryKey.
+// FindCompositePrimaryKeysByYZError retrieves multiple rows from 'CompositePrimaryKeys' as a slice of CompositePrimaryKey.
 //
 // Generated from index 'CompositePrimaryKeysByError3'.
-func FindCompositePrimaryKeysByZYError(ctx context.Context, db YORODB, e int8) ([]*CompositePrimaryKey, error) {
```
Comment (iwata) [inline test/testmodels/customtypes/compositeprimarykey.yo.go:377]: I ran it using Spanner emulator 1.5.15. I don't know the details, but the Storing column names returned by `INFORMATION_SCHEMA` are now sorted alphabetically. The name of this function has been changed due to this change.

File: cmd/root.go [hunk 2]
```diff
@@ -123,6 +124,7 @@ func setRootOpts(cmd *cobra.Command, opts *internal.ArgType) {
 	cmd.Flags().StringVarP(&opts.Out, "out", "o", "", "output path or file name")
 	cmd.Flags().StringVar(&opts.Suffix, "suffix", defaultSuffix, "output file suffix")
 	cmd.Flags().BoolVar(&opts.SingleFile, "single-file", false, "toggle single file output")
+	cmd.Flags().BoolVar(&opts.FilenameWithUnderscores, "with-underscores", false, "toggle underscores in file names")
```
Comment (shuheiktgw) [inline cmd/root.go:127]: To align with the other options such as the `suffix` flag, how about removing `with` and naming it `underscore`?

**Review by shuheiktgw — CHANGES_REQUESTED** [review]
Thanks for the PR! I left one comment, but it almost LGTM 👍

File: cmd/root.go (hunk 2 above)
Comment (iwata) [reply cmd/root.go]: fixed [2c06c11](https://github.com/cloudspannerecosystem/yo/pull/141/commits/2c06c119297a1da65c2f5bbf9cb730048df6289d)

**Review by shuheiktgw — APPROVED** [review]
Thank you! LGTM 👍

**Pull request merged**